core/src/main/kotlin/materialui/components/alert/AlertElementBuilder.kt

```diff
@@ -0,0 +1,25 @@
+package materialui.components.alert
+
+import kotlinx.html.Tag
+import kotlinx.html.TagConsumer
+import materialui.components.MaterialElementBuilder
+import materialui.components.alert.AlertProps
+import materialui.components.alert.enums.AlertVariant
+import materialui.components.appbar.enums.AlertSeverity
+import materialui.components.appbar.enums.AlertStyle
+import materialui.components.getValue
+import materialui.components.setValue
+import react.RClass
+
+class AlertElementBuilder<T : Tag> internal constructor(
```

Comment:
An implementation of the Alert component already exists in :kotlin-material-ui:lab.

https://github.com/subroh0508/kotlin-material-ui/blob/11ecec116c93bbc92009630cf144796dddd6f06e/lab/src/main/kotlin/materialui/lab/components/alert/AlertElementBuilder.kt#L18-L40